roles/common/tasks/main.yml

```diff
@@ -36,13 +32,17 @@
     dest=/home/jenkins/apache-maven-3.2.5-bin.tar.gz
     mode=0640
 
+- file: path=/home/jenkins/apache-maven-3.2.5-bin.tar.gz owner=jenkins group=jenkins mode=0644
```

Comment:
I'll remove it from get_url, but means I'll need to test it again.

On Wed, 25 Mar 2015 14:04 Sanne Grinovero <sanne@hibernate.org> wrote:

> I'd agree but because of a bug in Ansible that doesn't work..
>
> On Wed, 25 Mar 2015 12:04 Gunnar Morling <notifications@github.com> wrote:
>
>> In roles/common/tasks/main.yml
>> <https://github.com/hibernate/ci.hibernate.org/pull/16#discussion_r27114348>
>> :
>>
>> > @@ -36,13 +32,17 @@
>> >      dest=/home/jenkins/apache-maven-3.2.5-bin.tar.gz
>> >      mode=0640
>> >
>> > +- file: path=/home/jenkins/apache-maven-3.2.5-bin.tar.gz owner=jenkins group=jenkins mode=0644
>>
>> If you want to split it up, you can omit the "mode" attribute from
>> get_url. But for the sake of simplicity I'd simply give all three, i.e.
>> mode, owner and group directly for get_url and be done with it.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hibernate/ci.hibernate.org/pull/16/files#r27114348>.
>>
>